Mathlib/LinearAlgebra/CliffordAlgebra/Even.lean

```diff
@@ -181,8 +181,7 @@ private theorem fFold_fFold (m : M) (x : A × S f) : fFold f m (fFold f m x) = Q
     · rintro _ ⟨b, m₃, rfl⟩
       change f.bilin _ _ * (f.bilin _ _ * b) = Q m • (f.bilin _ _ * b)
       rw [← smul_mul_assoc, ← mul_assoc, f.contract_mid]
-    · change f.bilin m₁ m * 0 = Q m • (0 : A)  -- Porting note: `•` now needs the type of `0`
```

Comment:
My intent here was to use `change` to document what the proof is doing; it's primarily for human consumption.

Reply:
To that end I suppose `suffices foo by simpa` would be a non-smelly alternative to `change`

Reply:
I've changed this one.